### PR TITLE
Counter; to count the number of books in the database

### PR DESCRIPTION
**Pull request by mkhalil7625**
@ew3361zh

### CODE REVIEW

**Review by ew3361zh — CHANGES_REQUESTED** [review]
Nice work!
I have a couple things to suggest:
1. Make all the words in the menu option capitalized to match the formatting for the rest of the menu options (right now the word "total" is not capitalized).
2. Add an if/else block for the book_count function to account for grammar. So I would do something like:
if count == 1:
 print('There is 1 book in the database')
else:
print(f'There are {count} books in the database'}

Otherwise looks great!

**Review by ew3361zh — APPROVED** [review]
looks excellent and working well when I pulled it locally!

**Comment by mkhalil7625** [comment]
@ew3361zh Thank you for your feedback. I have incorporated your ideas and pushed the code into the branch, please review and let me know if you recommend other updates.
Thanks!

**Comment by ew3361zh** [comment]
@mkhalil7625 I did the "approve" changes in the review section and then merged the pull request. We should probably figure out the order of who's doing the merge so we don't accidentally double that up or do it too soon. Apologies if I jumped the gun. Nice work!

**Comment by mkhalil7625** [comment]
@ew3361zh No worries, I was writing my previous message when my daughter interrupted before I hit comment :)  so it appeared after your closing the request. 
I think if we always have the option to rollback our changes if needed or to create another branch for further updates

File: main.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def create_menu():
     menu.add_option('5', 'Show All Books', show_all_books)
     menu.add_option('6', 'Change Book Read Status', change_read)
     menu.add_option('7', 'Delete a Book', delete_book)
+    menu.add_option('8','Show Total Book Count', book_count)
```
Comment (claraj) [inline main.py:31]: Minor style note - add a space after the comma, similar to the lines before.

File: main.py [hunk 2]
```diff
@@ -68,14 +69,22 @@ def change_read():
...
         book_id = ui.get_book_id()
         book = store.get_book_by_id(book_id) 
         book.delete()
     except :
         print("Error: Book Not Found")
 
+
+def book_count():
+    count = store.book_count()
+    if count==1:
```
Comment (claraj) [inline main.py:83]: Style - spaces around operators like == is recommended
`if count == 1:`

**Review by claraj — COMMENTED** [review]
Small style notes

**Pull request merged**